content/solutions/docker.adoc

```diff
@@ -19,7 +19,29 @@ ____
 
 What this offers to Jenkins users is a means to isolate different jobs from one another, quickly clean a job's workspace or even dynamically deploy or schedule jobs with Docker containers to increase resource utilization and efficiency.
 
+== Getting Started
+
+* link:https://docs.docker.com/get-started/[Getting Started with Docker]
+* link:https://hub.docker.com/r/jenkins/jenkins[Jenkins CI/CD server in Docker]
+* link:https://medium.com/@gustavo.guss/quick-tutorial-of-jenkins-b99d5f5889f2[Jenkins+Docker Tutorial]
+
+== Demos
```

Comment:
Not sure we want to add that. I suspect all these demos are quite outdated.
CC @bitwiseman @dwnusbaum

Reply:
Yeah, we would probably prefer for users to use different approaches, but I don't know if there is any content that demonstrates the current best practices in this area.

Reply:
How about adding blogs related content, like the following:
`== Blogs

* link:https://www.jenkins.io/blog/2020/05/11/docker-windows-agents/#using-windows-docker-images/[Windows Docker agent]`